helpdesk_mgmt/tests/test_helpdesk_ticket.py

```diff
@@ -181,3 +183,101 @@ def test_ticket_without_team(self):
             }
         )
         self.assertEqual(self.new_stage, new_ticket.stage_id)
+
+    def test_helpdesk_ticket_multiple(self):
+        Ticket = self.env["helpdesk.ticket"]
+        Partner = self.env["res.partner"]
```

Comment:
**suggestion (non-blocking)**: All these could be put in the setUp once, instead of being repeated in every test.